### PR TITLE
editoast: make main script more homogeneous

### PR DESCRIPTION
**Pull request by Castavo**

### CODE REVIEW

**Review by flomonster — COMMENTED** [review]
We could return `CliError` instead of `println` command-line errors.

File: editoast/src/main.rs [hunk 1]
```diff
@@ -75,10 +75,10 @@ async fn main() {
         Ok(_) => (),
         Err(e) => {
             if let Some(e) = e.downcast_ref::<CliError>() {
-                println!("{e}");
+                eprintln!("{e}");
                 exit(e.exit_code);
             } else {
-                error!("{e}");
+                eprintln!("{e}");
```
Comment (hamz2a) [inline editoast/src/main.rs]: I prefer to leave `error!` to have the log information since this line should not be used by `CliError` errors

**Review by leovalais — APPROVED** [review]
LGTM.

**Review by hamz2a — APPROVED** [review]
LGTM.

**Review by Tguisnet — APPROVED** [review]
Nice

**Pull request merged**